### PR TITLE
[VA-17710] Update VBA "Schedule An Appointment Online" link

### PR DESCRIPTION
**Pull request by maxx1128**
## Summary

The VBA "Schedule an Appointment Online" link goes to the VAMC scheduling link. This fixes it by pointing it to VERA.

## Related issue(s)

- _Link to ticket created in va.gov-cms repo_
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17710

## Testing done

Local, see below screenshot

## Screenshots

Screen Shot 2024-04-19 at 7 37 09 PM

## What areas of the site does it impact?

VBA Services

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Comment by eselkin** [comment]
So, in the Service Locations branch we no longer use the cab service location liquid. Instead it uses the Service Location paragraph type that has the online scheduling link in there. That's where we should be updating I think.

**Comment by maxx1128** [comment]
@eselkin I updated the base branch to the service location paragraphs and made the change to the paragraph type instead 👍

**Comment by eselkin** [comment]
This needs to be conditioned on whether the service location is for VAMC or VBA. So a parameter needs to be added around that online scheduling link if typeOfLocation is VBA or not. Then if it is, display the updated link if not display the original.

**Comment by maxx1128** [comment]
@eselkin Added the extra conditional 👍

**Comment by eselkin** [comment]
I think `src/site/includes/vba_facilities/services.liquid` includes on lines 94 and 105 need:
`typeOfLocation = "vba"`

File: src/site/includes/vba_facilities/service_location.liquid [hunk 1]
```diff
@@ -73,10 +73,16 @@
     {% endfor %}
   {% endif %}
 
+  {% if typeOfLocation == "vba" %}
+    {% assign scheduleAppointmentLinkVBA = "https://va.my.site.com/VAVERA/s/" %}
+  {% else %}
+    {% assign scheduleAppointmentLinkVBA = "/health-care/schedule-view-va-appointments" %}
```
Comment (jilladams) [inline src/site/includes/vba_facilities/service_location.liquid:79]: This is a little confusing in terms of code being self-documenting, imo. We're creating a var for `scheduleAppointmentLinkVBA`, but using that same var when the link _isn't_ VBA, no?  Can this one be called something more generic like the other one below in PR (`scheduleAppointmentLink`) is?

**Comment by maxx1128** [comment]
@eselkin @jilladams Made both changes 👍

**Review by eselkin — APPROVED** [review]
LGTM! So I guess I had added that serviceLocationType variable before we used it for anything and didn't realize it was there when I created that other typeOfLocation variable. Sorry. Thanks for updating it.

**Pull request merged**